rpc/src/v1/helpers/signing_queue.rs

```diff
@@ -320,31 +301,30 @@ mod test {
 	}
 
 	#[test]
+	fn should_mark_future_as_done() {
+		let (sender, receiver) = oneshot::oneshot();
```

Comment:
we use tabs, not spaces see `.editorconfig`

Reply:
Side-effect of jumping from editor to editor, I guess; shouldn't happen anymore in the future.

I'll double-check spacing in all the files I've touched in this PR, thanks for noticing!